### DISCUSSION TITLE
Converting profile 7 to profile 8

### DISCUSSION THREAD

**Discussion by AvihaiV** [quietvoid/dovi_tool#316] [General]
Hi,

I have an MKV file with HDR and DV.
This is the DV info:
`DOVI configuration record: version: 1.0, profile: 7, level: 6, rpu flag: 1, el flag: 1, bl flag: 1, compatibility id: 6`

I am running the following comand to extract the audio and video:
`ffmpeg -i input.mkv -map 0:v:0 -c copy video.hevc -map 0:a:0 -c copy audio.eac3`

Then I am running the following command to convert the hevc file to profile 8:
`dovi_tool -m 2 convert video.hevc --output video_profile8.hevc`

Then adding all file to mp4 file:
`MP4Box -add video_profile8.hevc -add audio.eac3 output_profile8.mp4`

When checking the output file DV configuration it stays the same:
`DOVI configuration record: version: 1.0, profile: 7, level: 6, rpu flag: 1, el flag: 1, bl flag: 1, compatibility id: 6`

What am I missing here?

Thanks!

**Comment by quietvoid** [comment]
You didn't add `--discard` to the `convert` command so the EL is still present and MP4Box sees it as profile 7.

**Reply by AvihaiV** [reply]
Thank you!
So the correct command is:
`dovi_tool -m 2 convert video.hevc --output video_profile8.hevc --discard`

**Comment by AvihaiV** [comment]
I have another question, please.
When checking the original mkv file, I see all the metadata, but in the final mp4 I don't see anything, such as chapters and songs.

Is it possible that I am missing something along the way?

Thanks!

**Reply by quietvoid** [reply]
You're simply muxing the video and audio.
You need to find out how to retain the rest.

**Reply by AvihaiV** [reply]
Ok thanks!
